### PR TITLE
Fix payment jobs

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: app/services/cleanup-old-data/delete-old-payment-files.js [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
 const aws = new AWSHelper()
 
 module.exports = function (oldPaymentFiles) {
   oldPaymentFiles.forEach(async function (paymentFile) {
-    await aws.delete(paymentFile.Filepath)
+    try {
+      await aws.delete(paymentFile.Filepath)
+    } catch (error) {
+      log.error(`Failed to delete payment file: ${paymentFile.Filepath}`, error)
+    }
```
Comment (psoleckimoj) [inline app/services/cleanup-old-data/delete-old-payment-files.js:11]: Don't want a single problem file knocking out the whole batch.

File: start-schedule-payment-run.js [hunk 2]
```diff
@@ -5,24 +5,13 @@ const insertTask = require('./app/services/data/insert-task')
...
-        log.info('Cleanup Old Payment Files task created')
-      })
-  })
-  .then(function () {
-    insertTask('', '', '', taskTypes.GENERATE_PAYOUT_PAYMENTS)
-      .then(function () {
-        log.info('Payout Payment Generation task created')
-      })
-  })
+Promise.all([
```
Comment (psoleckimoj) [inline start-schedule-payment-run.js:10]: Reflects the daily file tasks now.

**Review by andrewrlee — APPROVED** [review]
LGTM

File: tmp/.gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```
Comment (andrewrlee) [inline tmp/.gitignore:2]: didn't know you could have .gitignore files in subdirectories! :-D 

Seen people use .gitkeep but [just found out](https://stackoverflow.com/a/7229996) thats not an actual thing more of a convention

Comment (psoleckimoj) [reply tmp/.gitignore:2]: I noticed it in these repos as I didn't realise that either 😄

**Pull request merged**